backend/app/repositories/base.py

```
@@ -0,0 +1,69 @@
+from fastapi.encoders import jsonable_encoder
+from sqlmodel import Session, SQLModel, select
+
+ModelType = TypeVar("ModelType", bound=SQLModel)
+CreateSchemaType = TypeVar("CreateSchemaType", bound=SQLModel)
+UpdateSchemaType = TypeVar("UpdateSchemaType", bound=SQLModel)
+
+
+class BaseRepository(Generic[ModelType, CreateSchemaType, UpdateSchemaType]):
+    """Base repository class for CRUD operations."""
+
+    def __init__(self, model: Type[ModelType]):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)